### ISSUE TITLE
Card as Checkbox + Cards as Form Component

### ISSUE THREAD

**Issue by anaclumos** [shuding/nextra#1683]
I like the Cards component. It would be very useful if we could

- set `onclick()`
- use as an input button & submit button

Probably something like this?

![IMG_7821](https://user-images.githubusercontent.com/31657298/227894018-50258d2a-033e-473c-bc5a-c8cc4f4e602f.jpeg)

**Comment by anaclumos** [comment]
Yeah, that's what I am doing for now. but suggesting as a change.

**Comment by anaclumos** [comment]
Yes it was for a simple signup form. My version is here: [hn.cho.sh](https://hn.cho.sh)

<img width="714" alt="Screenshot 2023-03-28 at 2 18 22 PM" src="https://user-images.githubusercontent.com/31657298/228369165-c177f497-2aa8-4c18-bae3-6ee31f14053b.png">

**Comment by SonMooSans** [comment]
Your design looks nice!

However, This feature isn't necessary or less used in documentation websites

Since many people would like to have more options to customise the component, creating their own one is more suitable for their requirements
